components/api/scrollable-menu/scrollable-section.js

```diff
@@ -1,10 +1,18 @@
-import React from "react";
+import React, { useRef, useEffect } from "react";
 import PropTypes from "prop-types";
 import Link from "next/link";
 
-const Section = ({ id, title, children }) => {
+const ScrollableSection = ({ id, title, addRef, children }) => {
```

Comment:
this interface is KO for this component : the complexity of addRef refers to internal behaviour and should not get appear outside of the component. A developer willing to use a Section does want to specify id, title and content. He does not want to specify addRef => this kind of complexity should be hidden.